lib/Parser.js

```diff
@@ -638,11 +641,14 @@ Parser.prototype.walkClassDeclaration = function walkClassDeclaration(statement)
 };
 
 Parser.prototype.walkSwitchCases = function walkSwitchCases(switchCases) {
-	switchCases.forEach(function(switchCase) {
-		if(switchCase.test)
+	for(var index = 0; index <= switchCases.length - 1; index++) {
```

Comment:
I'll update across the board. What kind of test would we expect in this situation. benchmarkCase?